week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -4,7 +4,7 @@
 function mood() {
   let isHappy = true;
 
-  if (isHappy) {
+  if (typeof isHappy !== "boolean") {
```

Comment:
It may not be very clear what's supposed to be fixed in this function but I don't think we want to check the type of value of `isHappy` (otherwise when `isHappy` is `false` we would return `"I am happy"`, which wouldn't be correct).

What else could we change in the function so that it makes sense (and makes the test pass)?